package.json

```diff
@@ -13,7 +13,7 @@
     "publish-workspaces": "yarn workspaces foreach --verbose --no-private npm publish --access public --tolerate-republish"
   },
   "engines": {
-    "node": ">=18.0.0"
+    "node": ">=20.0.0"
```

Comment:
Updated because the `recursive` option used in the icons script requires node 20. Also [we're ~halfway through the maintenance period for 18](https://nodejs.org/en/about/previous-releases#release-schedule) so it seems recommendable for anyone on older versions to update anyway.